ansible/environments/docker-machine/group_vars/all

```diff
@@ -3,6 +3,7 @@ config_root_dir: /Users/Shared/wskconf
 whisk_logs_dir: /Users/Shared/wsklogs
 docker_registry: ""
 docker_dns: ""
+bypass_pull_for_local_images: true
```

Comment:
@csantanapr what do you think about making this true for the local envs?

Reply:
Yes make it true for local 👍